### PR TITLE
Make sure that EnableOpenSource is installed on VS 2015

### PR DESCRIPTION
**Pull request by dungpa**
I ran EnableOpenSource from fsharp4 branch. The vsix was installed in VS 2013, which is misleading.

The PR changes the target installation to VS 2015 and includes the License file as part of the vsix.

### CODE REVIEW

**Comment by dsyme** [comment]
LGTM

**Comment by latkin** [comment]
@KevinRansom I think you are working on something maybe in this area?

**Comment by KevinRansom** [comment]
These are both good changes.

**Comment by latkin** [comment]
@KevinRansom Yes they are good.  I was checking if you want to accept this PR or are these changes already covered by what you are doing?

File: .gitignore [hunk 1]
```diff
@@ -94,3 +94,8 @@ tests/fsharpqa/Source/CodeGen/EmittedIL/QueryExpressionStepping/Utils.dll
 tests/fsharpqa/Source/CodeGen/EmittedIL/ComputationExpressions/ComputationExprLibrary.dll
 
 *.csproj.user
+
+*.ide
```
Comment (forki) [inline .gitignore]: Why is the gitignore changed?

Comment (dungpa) [inline .gitignore]: VS 2015 generates temporary files for projects inside `vsintegration` folder. These files haven't been git-ignored yet.

**Pull request closed without merging**